.github/workflows/ci.yml

```diff
@@ -36,3 +36,35 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    env:
+      REGISTRY: "quay.io"
+      REGISTRY_REPO: "igorshubovych"
+      IMAGE_NAME: "markdownlint-cli"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1.2.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1.5.1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1.10.0
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ secrets.REGISTRY_USERNAME }}
```

Comment:
Who provides these secrets and maintains them?

Reply:
This would be on the repo owner(s). They just need to be created and maintained by creating secrets in the repo settings.

Reply:
I do not (currently) have Docker credentials.

Reply:
Gotcha. At least from what I was able to gather, I don't think there is an existing docker/quay/etc. repository for this. So it would be as simple as just creating one? But I may be wrong here.